src/main/java/org/generationcp/ibpworkbench/security/WorkbenchAuthenticationSuccessHandler.java

```diff
@@ -49,10 +53,11 @@ public void onAuthenticationSuccess(final HttpServletRequest request, final Http
 		}
 
 		final WorkbenchUser user = retrieveUserFromAuthentication(authentication);
+		final Optional<ReleaseNote> releaseNote = this.releaseNoteService.shouldShowReleaseNote(user.getPerson().getId());
```

Comment:
It seems to me optional is kind of unnecessary here, boolean, true or false feels more appropriate. It should be possible to map all possible states of the system to `shouldShowReleaseNote()` -> either true or false, even when there are scenarios where release note info might not exist (null).

Reply:
Done!